src/encoded/tests/data/inserts/sc_atac_alignment_quality_metric.json

```diff
@@ -0,0 +1,40 @@
+[
+    {
+        "status": "released",
+        "step_run": "63b1b347-f008-4103-8d20-0e12f54d1882",
+        "uuid": "b76ed89f-9d40-44c4-8535-5185232eb20f",
+        "assay_term_name": "single-nucleus ATAC-seq",
+        "quality_metric_of": ["ENCFF789ARC"],
+        "lab": "/labs/john-stamatoyannopoulos/",
+        "award": "/awards/U54HG007010/",
+        "with_itself": 9382,
+        "with_itself_qc_failed": 741,
+        "usable_fragments": 1029,
+        "properly_paired_reads": 830,
+        "properly_paired_reads_qc_failed": 95,
+        "paired_reads": 8723,
+        "paired_reads_qc_failed": 64,
+        "pct_properly_paired_reads": 0.96,
+        "mapped_reads": 5390,
+        "mapped_reads_qc_failed": 89,
+        "pct_mapped_reads"
+        "singletons": 3890,
+        "singletons_qc_failed": 90,
+        "read1": 4540,
+        "read1_qc_failed": 67,
+        "read2": 4540,
+        "read2_qc_failed": 67,
+        "total_reads": 9080,
+        "total_reads_qc_failed": 42,
+        "duplicate_reads": 990,
+        "duplicate_reads_qc_failed": 301,
+        "pct_singletons": 0.89,
+        "diff_chroms": 599,
+        "diff_chroms_qc_failed": 239,
+        "non_mito_reads": 200,
+        "mito_reads":2000,
+        "frac_mito_reads": 0.3,
+        "mito_stats": "ENCFF150HADvENCFF315YLZ.IDRv2.pooled_common_IDRv2.narrowPeak.png",
+        "samstats": "ENCFF150HADvENCFF315YLZ.IDRv2.pooled_common_IDRv2.narrowPeak.png",
```

Comment:
A general comment about all of the test inserts - the file format for the attachment is image/png in your examples - but I think defined in the schemas it is pdf. Should you be able to accept either pdf or png?